tests_thien/pages/dashboard/main_page.py

```diff
@@ -0,0 +1,10 @@
+from selenpy.element.base_element import BaseElement
+
+class DashBoardMainPage():
+    _title_page_name = BaseElement("xpath=//title")
+    
+    def __init__(self):
+        pass
+    
+    def check_main_page_appears(self):
```

Comment:
The name should be `is_at()` or `is_page_played()`

Reply:
updated to `is_main_page_displayed()`